tests/test_autonomy/test_cli/test_scaffold_fsm.py

```diff
@@ -47,10 +47,11 @@
 fsm_specifications = list(VALORY_SKILLS_PATH.glob("**/fsm_specification.yaml"))
 
 
-class BaseScaffoldFSMTest(AEATestCaseEmpty):
+class BaseScaffoldFSMTest(AEATestCaseMany):
```

Comment:
@Karrenbelt the one you used already sets up an agent. Here we want to do this for each test method so we need to use the other class...

helps to read the code.

Reply:
so `Empty` is not empty :upside_down_face: